packages/gatsby-cli/src/create-cli.js

```diff
@@ -40,6 +45,11 @@ function buildLocalCommands(cli, isLocalSite) {
         `gatsby`,
         `package.json`
       ))
+      try {
+        setDefaultTags({ installedGatsbyVersion: packageInfo.version })
+      } catch (e) {
+        // ignore
```

Comment:
when does this throw an error? unsure if we need to try catch

Reply:
We could get around with a couple of iffs too.

Reply:
no biggy just can't think of any way this would throw

Reply:
yeah, I think it's just me playing extra safe to minimize places where the instrumentation would cause any issues